_books/ion-1-1/src/modules.md

```diff
@@ -22,10 +22,10 @@ The interface to a module consists of:
 
 The spec version is external to the module body and the precise way it is determined depends on the type of module being defined. This is explained in further detail in [Module Versioning](#module-versioning).
 
-The exported symbol array is denoted by the `symbol_table` clause of a module definition, and
+The exported symbol array is denoted by the `symbols` clause of a module definition, and
 by the `symbols` field of a shared symbol table.
 
-The exported macro array is denoted by the module’s `macro_table` clause, with addresses
+The exported macro array is denoted by the module’s `macros` clause, with addresses
 allocated to macros or macro bindings in the order they are declared.
```

Comment:
I suggest swapping the order of these two statements to align with the order of the clauses. It would be good to explicitly note that new symbols defined in the `symbols` clause are not yet available in the preceding `macros` clause.